qutip/floquet.py

```diff
@@ -833,7 +833,7 @@ def floquet_markov_mesolve(R, ekets, rho0, tlist, e_ops, f_modes_table=None,
         if not r.successful():
             break
 
-        rho.data = vec2mat(r.y)
```

Comment:
@nonhermitian would you have expected this `vec2mat` to return a `fast_csr_matrix`